lambdas/utils/decorators/handle_lambda_exceptions.py

```diff
@@ -0,0 +1,25 @@
+from typing import Callable
+
+from utils.exceptions import LambdaException
+from utils.lambda_response import ApiGatewayResponse
+
+
+def handle_lambda_exceptions(lambda_func: Callable):
+    """A decorator for lambda handler.
+    Catch our custom LambdaException and return an API gateway response according to the exception
+
+    Usage:
+    @handle_lambda_exceptions
+    def lambda_handler(event, context):
+        ...
+    """
+
+    def interceptor(event, context):
+        try:
+            return lambda_func(event, context)
+        except LambdaException as e:
```

Comment:
Where are we raising LambdaException? I thought we're just going to use it as a base class